src/Common/src/Interop/Unix/libssl/StreamSizes.cs

```diff
@@ -12,9 +12,17 @@ internal class StreamSizes
 
         internal StreamSizes()
         {
+            // Since the header and trailer values aren't being calculated anyways just
+            // report them as 0.
+            int ignoredHeader;
+            int ignoredTrailer;
+
+            header = 0;
+            trailer = 0;
+
             Interop.Ssl.GetStreamSizes(
-                out header,
-                out trailer,
+                out ignoredHeader,
+                out ignoredTrailer,
                 out maximumMessage);
```

Comment:
Well, that would cause an n/n-1 ABI compatibility problem that I was hoping to avoid for the RC2 fix. But, yeah, we can slim this down (with a different name) for RTM.